### PR TITLE
Tweak: Remove deprecated schemes area [ED-9386]

### PR DESCRIPTION
**Pull request by KingYes**

### CODE REVIEW

File: core/upgrade/upgrades.php [hunk 1]
```diff
@@ -945,14 +787,6 @@ public static function _v_3_4_8_fix_font_awesome_default_value_from_1_to_yes() {
 		}
 	}
 
-	public static function _v_3_5_0_remove_old_elementor_scheme() {
```
Comment (Nevoss) [inline core/upgrade/upgrades.php:948]: Why not keep it? if users comes from old version so the schema will be removed

File: includes/plugin.php [hunk 2]
```diff
@@ -754,7 +739,6 @@ private function init_components() {
 			$this->wordpress_widgets_manager = new WordPress_Widgets_Manager();
 			$this->admin = new Admin();
 			$this->beta_testers = new Beta_Testers();
-			new Elementor_3_Re_Migrate_Globals();
```
Comment (Nevoss) [inline includes/plugin.php:757]: Maybe we still need it?

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: includes/plugin.php (hunk 2 above)
Comment (KingYes) [reply includes/plugin.php:757]: It's used with a deprecation functions.

**Review by Nevoss — DISMISSED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please keep the PHP public API, so, old plugins and themes that calling [`Elementor::instance()->schemes_manager`](https://github.com/search?type=code&q=-%3Eschemes_manager) will not throw a fatal error.

**Review by arielk — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**